### DISCUSSION TITLE
Where do i download this

### DISCUSSION THREAD

**Discussion by FroofyFroof** [adieyal/sd-dynamic-prompts#646] [General]
There0's only a go to file button but no download

**Comment by Picometer12** [comment]
To the right of _Go to file_ is a green button labeled _Code_. You can click that and then download zip. 

But if you want to load this into Automatic1111, you can do it more easily and directly by copying the URL (also under the code button) and pasting into Extension -> Install from URL tab -> URL for extension's git repository
